example_configs/zulip.md

```diff
@@ -0,0 +1,133 @@
+# Configuration for Zulip
+
+Zulip combines the immediacy of real-time chat with an email threading model.
+
+Their ldap-documentation is here: [zulip.readthedocs.io](https://zulip.readthedocs.io/en/stable/production/authentication-methods.html#ldap-including-active-directory)
+
+Zulip has two installation methods, either by running the recommended installer or by docker/podman compose.
+The way how the service is configured differs depending the installation method, so keep in mind you will only need one of the following examples.
+
+> Important info  
+> The available/configured userdata will be automatically imported at the first login.  
+> If you want to import that data before the user logs in for the first time or  
+> if you want to keep the data in sync with LLDAP you need to trigger the import by hand (or via cronjob).  
+> `/home/zulip/deployments/current/manage.py sync_ldap_user_data`
+
+## Container based configuration
+The following configuration takes place in the environment section of your compose-file.
+
+1) Enable the LDAP authentication backend  
+Find the line`ZULIP_AUTH_BACKENDS: "EmailAuthBackend"` and change it to `ZULIP_AUTH_BACKENDS: "ZulipLDAPAuthBackend,EmailAuthBackend"`.
+
+2) Configure how to connect with LLDAP  
+The user specified in `SETTING_AUTH_LDAP_BIND_DN` is used to querry data from LLDAP.  
+Zulip is only able to authenticate users and read data via ldap it is not able to write data or change the users password.  
```

Comment:
I thought we could modify user attributes, like the name or avatar? That requires admin permissions

Reply:
Zulip imports all available user attributes when the user is created (first login or manual sync).
After that the users can change their attributes however they like.
But those changes are only made in Zulip they are not push back to the ldap server.

If someone triggers the command to resync all existing users (usually a cronjob) everything a user has changed by themself is overwritten again. It is only a one-way-sync.

Reply:
Hmm, I think we should call that out in the config, in the section where we allow editing the name and avatar.
Otherwise, people are going to be (badly) surprised.